src/rest/app.py

```diff
@@ -0,0 +1,178 @@
+import json
+import logging
+import time
+import uuid
+from datetime import datetime
+
+from fastapi import (Depends, FastAPI, File, Form, HTTPException, Security,
+                     UploadFile)
+from fastapi.responses import JSONResponse, PlainTextResponse
+from fastapi.security.api_key import APIKeyHeader
+from pydantic import BaseModel
+from pydub import AudioSegment
+from starlette.status import HTTP_401_UNAUTHORIZED
+
+from src.helper.config import CONFIG
+from src.helper.data_handler import DataHandler
+from src.helper.types.transcription_status import TranscriptionStatus
+
+LOGGER = logging.getLogger(__name__)
+DATA_HANDLER = DataHandler()
+
+
+class TranscriptionData(BaseModel):
```

Comment:
Move this to a separate file maybe? I think this makes this file more convoluted than it needs to be.

Reply:
this still stands